### ISSUE TITLE
Style property is null on IHtmlElement when trying to set CSS properties

### ISSUE THREAD

**Issue by manigandham** [AngleSharp/AngleSharp#90]
This used to work fine, now the Css helper method is trying to access the Style property (ICssStyleDeclaration) on IHtmlElement which is null. Is this a bug or did the API change for setting a inline style property?

```
new HtmlParser("some html...").Parse().QuerySelectorAll("img").Css("width","100%");
```

**Comment by FlorianRappl** [comment]
Hm sounds like a bug. At least that was not intentional. I will look into that!

**Comment by FlorianRappl** [comment]
Alright so I checked and I realized I didn't look at your code in detail. What you are missing is actually CSS styling. You use the parser directly, which is fine, but you do not supply a custom configuration. By default everything that is not required for (HTML) parsing (e.g., CSS) is disabled.

For example:

```
var config = new Configuration().WithCss();
new HtmlParser("...", config).Parse().QuerySelectorAll("img").Css("width","100%");
```

This should then work. Hope this helps you!
